yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file is not needed as we don't have `package.json` in the root

Reply:
Removed with [commit](https://github.com/podkrepi-bg/frontend/pull/278/commits/d72ade00d9c390f0756265dc65d1fb8e319bf89d)